renku/ui/cli/session.py

```diff
@@ -231,7 +231,7 @@ def stop(session_name, stop_all, provider):
     help="Session provider to use.",
 )
 def open(session_name, provider):
-    """Stop a interactive sessions."""
+    """Open a interactive sessions."""
```

Comment:
```suggestion
    """Open an interactive session."""
```

Reply:
Ah, I've been lazy with fixing the typos :sweat_smile: 
I added a commit fixing also the other similar sentences. Thanks for the review!